### DISCUSSION TITLE
Gltf help required

### DISCUSSION THREAD

**Discussion by suprohub** [bevyengine/bevy#17932] [Q&A]
**how i can get single node with parents instead of list with random nodes?**

i have something like this
[ node1, node3 -> node2, node2, node3, node1 -> node3 ]

but i want this
node1 -> node3 -> node2

**can i get this? or i need to write code for solving whis?**

And answering your questions why me need this.
I creating a trimesh generator, but its generate wrong colliders, because mesh needs to inherit transform from parents.

**Comment by suprohub** [comment]
Але помогайте давайте че вы молчите, я с этой фигней уде несколько дней мучаюсь ахпхп

**Comment by HyperCodec** [comment]
> Але помогайте давайте че вы молчите, я с этой фигней уде несколько дней мучаюсь ахпхп

Please be patient, they prioritize issues that are bugs or features within the engine itself over questions. In the future you should join their discord at https://discord.gg/bevy, which has help channels dedicated to these types of questions and will get you support much faster.

As for your question, could you be more specific? (although I'm probably not very well qualified to answer it because I don't typically mess with glTF)

**Comment by BenjaminBrienen** [comment]
README.md:

> * **[GitHub Discussions](https://github.com/bevyengine/bevy/discussions):** The best place for questions about Bevy, answered right here!

**Comment by BenjaminBrienen** [comment]
It would help if you showed code or RON instead of whatever syntax that is.
